### PR TITLE
Remove removed `upgrade` commands

### PR DESCRIPTION
**Pull request by pselle**
Delete commands that are no longer in Terraform and have been removed for at least a full release cycle.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -328,18 +328,6 @@ func initCommands(
 		// Plumbing
 		//-----------------------------------------------------------
 
-		"0.12upgrade": func() (cli.Command, error) {
-			return &command.ZeroTwelveUpgradeCommand{
-				Meta: meta,
-			}, nil
-		},
-
-		"0.13upgrade": func() (cli.Command, error) {
```
Comment (alisdair) [inline commands.go:337]: There's another mention of these in the `HiddenCommands` map later in this file that we could remove.

**Review by alisdair — APPROVED** [review]
Looks good to me! I think it makes sense to leave the website docs around for now, since we don't yet have versioned docs.

File: commands.go (hunk 1 above)
Comment (pselle) [reply commands.go:337]: Thank you!

**Comment by pselle** [comment]
@alisdair Thanks! Yes that was my thought on the website as well. 🙏 Versioned docs one day.

**Pull request merged**